src/04-meet-your-hardware/README.md

```diff
@@ -8,40 +8,79 @@ Let's get familiar with the hardware we'll be working with.
 <img title="F3" src="../assets/f3.jpg">
 </p>
 
-We'll refer to this board as "F3" throughout this book.
-
-What does this board contain?
-
-- A STM32F303VCT6 microcontroller. This microcontroller has
-  - A single core ARM Cortex-M4F processor with hardware support for single precision floating point
-    operations and a maximum clock frequency of 72 MHz.
-
-  - 256 KiB of "Flash" memory. (1 KiB = 10**24** bytes)
-
-  - 48 KiB of RAM.
-
-  - many "peripherals": timers, GPIO, I2C, SPI, USART, etc.
-
-  - lots of "pins" that are exposed in the two lateral "headers".
-
-  - **IMPORTANT** This microcontroller operates at (around) 3.3V.
-
-- An [accelerometer] and a [magnetometer][] (in a single package).
+We'll refer to this board as "F3" throughout this book. Here are some of the
+many components on the board:
+
+- A number of LEDs, including the eight aligned in a "compass" formation.
+- Two buttons.
+- Two USB ports.
+- An [accelerometer].
+- A [magnetometer].
+- A [gyroscope].
```

Comment:
I appreciate the list isn't meant to be exhaustive but since you later say "of _these_ components, the most important is the microcontroller", perhaps the microcontroller should be mentioned in this list, perhaps even at the top.

Reply:
Good point. Added in a22fa6017d9437c46f51172f161761e802d6af45.